File: web3rpc/rpc-specs/api/code-samples/javascript/klay/transaction/getRawTransactionByBlockHashAndIndex.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const blockHash = '0x4c4cbf242a80183d2ea2daf047c578d5fc89c0b14c4262606c8b6bb0b36715be'
    const transactionIndexPosition = '0x0'
    
    sdk.klay.getRawTransactionByBlockHashAndIndex(blockHash, transactionIndexPosition, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/api/code-samples/python/eth/others/getRawTransactionByBlockHashAndIndex.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0x4c4cbf242a80183d2ea2daf047c578d5fc89c0b14c4262606c8b6bb0b36715be"
index = "0x0"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_raw_transaction_by_block_hash_and_index(blockHash, index)

print(eth_response)
